python_ta/checkers/inconsistent_or_missing_returns_checker.py

```diff
@@ -25,6 +25,17 @@ class InconsistentReturnChecker(BaseChecker):
             "Used when a function does not have a return statement and whose return type is not None",
         ),
     }
+    options = (
+        (
+            "z3",
+            {
+                "default": False,
+                "type": "yn",
+                "metavar": "<y or n>",
+                "help": "Only check for logically feasible nodes based on edge z3 constraints",
```

Comment:
Reword this to "Use Z3 to restrict control flow checks to paths that are logically feasible" (and change throughout)